### ISSUE TITLE
Logs are not getting attached while executing Pabot with ReportPortal

### ISSUE THREAD

**Issue by deekhare911** [reportportal/agent-Python-RobotFramework#195]
**Describe the bug**
Logs are not getting attached while executing Pabot with ReportPortal

**Steps to Reproduce**
1. Create Launch using API:
`import requests
import json

url = "https://demo.reportportal.io/api/v1/:projectName/launch"

payload = json.dumps({
  "attributes": [
    {
      "key": "string",
      "system": False,
      "value": "string"
    }
  ],
  "description": "string",
  "mode": "DEFAULT",
  "name": "string",
  "rerun": True,
  "rerunOf": "string",
  "startTime": "2024-06-21T07:32:35.273Z"
})
headers = {
  'Content-Type': 'application/json',
  'Accept': '*/*',
  'Authorization': 'bearer ' + rp_token
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)`

2. Execute Pabot command: (RP_LAUNCH_UUID - we are getting from Step1)
`pabot --processes 3 --testlevelsplit --listener robotframework_reportportal.listener -v headless:True -v URL:https://helloworld.com/ -v Username:auto@cloud.com -v Password:*** -v ENV:QA -v CLIENT:Internal --exclude platform:Windows -v RP_LAUNCH:Demo_NewRPTest -v RP_LAUNCH_UUID:10171752-9d5e-4345-a26d-*** -v RP_ENDPOINT:https://reportportal.artegocloud.com -v RP_API_KEY:*** -v RP_PROJECT:Internal **-v RP_ATTACH_LOG:True -v RP_ATTACH_XUNIT:True -v RP_ATTACH_REPORT:True** --xunit xunit_output.xml --outputdir /testoutputs --include SmokeTestsANDENV:QAANDCLIENT:Internal TestCases/*`

3. Finish Launch using API:
`import requests
import json

url = "https://demo.reportportal.io/api/v1/:projectName/launch/:launchId/finish"

payload = json.dumps({
  "attributes": [
    {
      "key": "string",
      "system": False,
      "value": "string"
    }
  ],
  "description": "string",
  "endTime": "2024-06-21T07:32:35.273Z",
  "status": "PASSED"
})
headers = {
  'Content-Type': 'application/json',
  'Accept': '*/*',
  'Authorization': 'bearer ' + rp_token
}

response = requests.request("PUT", url, headers=headers, data=payload)

print(response.text)`

**Expected behavior**
This should create a single launch - with logs attached - because we have given **-v RP_ATTACH_LOG:True -v RP_ATTACH_XUNIT:True -v RP_ATTACH_REPORT:True**

**Actual behavior**
Single launch is getting created - but logs are not attached

**Package versions**
robotframework ==7.0
robotframework-pabot==2.17.0
reportportal-client==5.5.4
robotframework-reportportal==1.0.0
Include version info of the following packages: reportportal-client, robotframework-reportportal

**Additional context**
Please let me know if I'm missing something here.

![image](https://github.com/user-attachments/assets/b134fdcb-cc89-4abe-9e25-6730b0cdc029)

![image](https://github.com/user-attachments/assets/77bc3b11-9b1b-43ed-bdf6-dfa673844d90)

![image](https://github.com/user-attachments/assets/a40e76b5-c626-4cc1-a7f4-deaccd7b3787)

**Comment by HardNorth** [comment]
@deekhare911 No, this is drawback of using Pabot. I just don't have any logs in runtime.

**Comment by HardNorth** [comment]
@deekhare911 I don't see any issue on my end, logs appear correctly:
![Screenshot 2024-09-23 at 15 48 32](https://github.com/user-attachments/assets/04cfbb5a-e62d-4a76-97c0-b32a96cb5a91)
So this should be some kind of configuration issue on your side. I upgraded all dependencies in our examples and implement Pabot run script for to ease configuration for users:
https://github.com/reportportal/examples-python/blob/master/robotframework/run-pabot.sh

Hope this will help.
